packages/serverless-contracts/src/contracts/SQS/utils/serializeMessageAttributes.ts

```diff
@@ -0,0 +1,61 @@
+import { SendMessageRequest } from '@aws-sdk/client-sqs';
+import { MessageAttributeValue } from '@aws-sdk/client-sqs/dist-types/models/models_0';
+import { JSONSchema } from 'json-schema-to-ts';
+
+import { SQSContract } from '../sqsContract';
+
+const serializeMessageAttribute = (
+  attributeValue: unknown,
+  attributeSchema: JSONSchema,
+): MessageAttributeValue => {
+  if (typeof attributeSchema === 'boolean') {
+    throw new Error('Invalid contract messageAttributesSchema');
+  }
+  const { type } = attributeSchema;
+
+  switch (type) {
+    case 'string':
+      return { StringValue: attributeValue as string, DataType: 'String' };
+    case 'number':
+      return {
+        StringValue: (attributeValue as number).toString(),
+        DataType: 'Number',
+      };
+    default:
+      throw new Error(
+        'Invalid messageAttributesSchema. Only string and number are currently supported',
+      );
+  }
+};
+export const serializeMessageAttributes = <
+  MessageAttributes extends Record<string, unknown>,
+  Contract extends SQSContract,
+>(
```

Comment:
So this only supports a single depth of objects? If that is the case, maybe this should be mentioned in the docs